.github/workflows/auto-merge-bot.yml

```diff
@@ -0,0 +1,19 @@
+on: [pull_request]
+
+jobs:
+  auto_merge_bot:
+    runs-on: ubuntu-latest
+    name: EIP Auto-Merge Bot
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js Environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: auto-merge-bot
+        uses: alita-moore/EIP-Bot@1.1.1 # v1.1.1 does not have the ability to merge and throws and error on non-merge like posting comment
+        continue-on-error: true
+        id: auto-merge-bot
+        with:
+          GITHUB-TOKEN: ${{ secrets.TOKEN }} 
```

Comment:
If this isn't set, what will happen?  Will it fail and flag the build as failing (thus preventing merge), or will it fail and turn the build yellow but not block merge?  Perhaps this depends on settings on this repository (whether an action is required or not)?

Reply:
It will fail but the ability to merge is independent of that / the auto-merge bot status can be made optional